README.md

```diff
@@ -55,9 +55,9 @@ The backlinks calculation is costly to your computers, so it is recommended to t
 $ ./main.py -l search {URI-R} -e cdGetBacklinks
 ```
 
-## How to add your module:
+## How to add your module
 
-Name your module main script as cdGet\<Module name\>.py
+Name your module's main script as `cdGet\<Module name\>.py`. If your module relies on extra code in a different folder you may bring this directory into the modules directory, the CarbonDate library will ignore subfolders while loading modules. If the extra code is not in a subfolder, after copying it to the `./modules` folder, add the file names into the config file, under `SystemUtility` field.
```

Comment:
`cdGet\<Module name\>.py` => `cdGet<ModuleName>.py` (note the camelCase module name without a space, also escaping is unnecessary.)

Should "under `SystemUtility` field" be "under the `SystemUtility` field"?